plugins/wfp/management/commands/wfp_etl_Under5.py

```diff
@@ -10,19 +10,38 @@
 
 
 class Under5:
-    def compute_gained_weight(self, initial_weight, current_weight):
+    def compute_gained_weight(self, initial_weight, current_weight, duration):
         weight_gain = 0
+        weight_loss = 0
+
+        weight_difference = 0
         if initial_weight is not None and current_weight is not None:
-            weight_gain = float(current_weight) - float(initial_weight)
-        return weight_gain
+            initial_weight = float(initial_weight)
+            current_weight = float(current_weight)
+            weight_difference = round((current_weight - initial_weight), 4)
+
+            if weight_difference >= 0:
+                if duration > 0 and current_weight > 0:
+                    weight_gain = round((weight_difference / (initial_weight * float(duration))), 4)
+            elif weight_difference < 0:
+                weight_loss = abs(weight_difference)
```

Comment:
Is it normal that weight_gain seems to be a percentage and weight loss the exact value?

Reply:
yes, the formula to calculate the weight gain has been shared by Sofia. It's an average of weight gained by the beneficiary during the journey(from admission in a program to exit it). The weight loss was not supposed to be calculated, but I proposed to calculate it also with that formula and Sofia was agree!

You can find more details in the last comment in: https://dev.azure.com/worldfoodprogramme/CODA2/_workitems/edit/241569